src/components/picker/index.js

```diff
@@ -337,7 +339,13 @@ class Picker extends PureComponent {
         onCancel: this.cancelSelect
       };
 
-      return renderCustomModal(modalProps);
+      return (
+        <>
```

Comment:
What is the use of the empty tag?

Reply:
It is shorter way to wrap JSX with React.Fragment, so instead of doing
```
<React.Fragment>
....
</React.Fragment>
```
you can just do 
```
<>
....
</>
```